src/app/profile/update/update.component.scss

```diff
@@ -1,10 +1,80 @@
 @import '../../../assets/stylesheets/base';
 @import '../../../assets/stylesheets/shared/layout';
+
 .checkbox input[type="checkbox"] {
   margin-left: 0;
   margin-right: 5px;
   position: relative;
 }
-.fields-status-pf { margin-top: -5px; }
-.getting-started { padding-bottom: 20px; }
-.margin-top-40 { margin-top: 40px; }
+
+.fields-status-pf {
```

Comment:
Will do. However, it doesn't make for good readability. In the event of an unlikely error, it's only one line number off.